bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/Asterism.scala

```diff
@@ -84,6 +85,8 @@ trait Asterism {
   /** Construct a copy of this Asterism with cloned SPTargets (necessary because they are mutable). */
   def copyWithClonedTargets: Asterism
 
+  // Get the type of this asterism as an AsterismType.
+  def asterismType: AsterismType
```

Comment:
I also forced `Asterism`s to return their `AsterismType`, since I think not only will this be useful, but it will also force any further added subclasses of `Asterism` to define this and thus update `AsterismType`.

Reply:
Should it be a type param instead?